src/app/tests/DeleteObjects.test.ts

```diff
@@ -0,0 +1,34 @@
+import {V} from "Vector";
+import {Component}     from "core/public/api/Component";
+
+import {CreateCircuit} from "digital/public";
+
+import "./Extensions";
+
+
+describe("DeleteObjects", () => {
+    test("Delete Component", () => {
+        const circuit = CreateCircuit();
+
+        const c = circuit.placeComponentAt(V(0, 0), "ANDGate");
+
+        expect(circuit.getObjs()).toHaveLength(4); // 2 inputs, 1 output, 1 component
+        expect(c.pos).toEqual(V(0, 0));
+
+        let cArray: Component[] = [c];
+        circuit.deleteObjs(cArray);
+        expect(cArray[0]).toEqual(undefined);
```

Comment:
This is testing if it gets removed from the array, we need to make sure that it was actually deleted from `circuit`